languages/burmese.md

```diff
@@ -48,3 +48,11 @@ supported_apis:
 
 ---
 
+## Character encoding
+
+Burmese characters can be represented in Unicode or Zawgyi.
+Zawgyi is a variation on Unicode which interprets some code points differently but otherwise encodes code points in the same way with UTF-8 and other encodings.
+If Zawgyi text is interpreted as Unicode, some characters will appear garbled.
+
+When building training data for machine translation from web text, the data will include a mixture of Burmese Unicode and Burmese Zawgyi.
+This can lead to fragmentation of some of the vocabulary unless the data is first normalized.
```

Comment:
Is mixed encoding perhaps desirable when going *from* Burmese to e.g. English, but not from e.g. English *to* Burmese?

Reply:
I hadn't considered that... I _think_ it'd be desirable yes, because end users may not know the encoding of their text so it'd be good to support both.